docs/concepts/07_dip/04_dapp_developer.md

```diff
@@ -0,0 +1,29 @@
+---
+id: dapp-developer
+title: Dapp developer
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Decentralized Identity Provider (DIP) SDK helps Dapp developers build DIP functionality into their apps.
+
+## Installation
+
+Add the SDK as a dependency:
+
+```bash npm2yarn
+npm install @kiltprotocol/dip-sdk
+```
+
+Import the SDK into your code:
+
+```typescript
+import { * } from '@kiltprotocol/dip-sdk'
```

Comment:
You can either `import * from '@kiltprotocol/dip-sdk'` or `import * as DipSdk from '@kiltprotocol/dip-sdk'`.

```suggestion
import * from '@kiltprotocol/dip-sdk'
```